functional_test.go

```diff
@@ -1618,6 +1618,8 @@ func TestHealthCheck(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), clock.Second*15)
 	defer cancel()
 	require.NoError(t, cluster.Restart(ctx))
+	// wait for instances to come online
```

Comment:
Would prefer if we could poll with `testutil.UntilPass()` for healthy.  A fixed delay may still be flaky on slow GHA servers.

Reply:
updated